### DISCUSSION TITLE
How to remove the need for the user to write "/start" after each code update and bot restart

### DISCUSSION THREAD

**Discussion by Abusher** [telegraf/telegraf#1495] [Q&A]
Hello,

In my current implementation, after each code changes and restart, User should write "/start" command again
to make bot work and handle user commands.

Can I somehow avoid this mandatory "/start" for user after each code update?

**Comment by kdabwl** [comment]
Clear history, it does belong to outdated code, then /start is presented automatically as if bot taken into chat first time.
